CHANGELOG_UNRELEASED.md

```diff
@@ -0,0 +1,5 @@
+## [0.2.0] - 2022-02-08
+
+### Added
+* Added method navigateToLocation(floor, lat, lng) on SitumMapLibrary to navigate towards a coordinate on the map
+specified by latitude and longitude.
```

Comment:
...on SitumMapLibrary to navigate to a location in the current building. The location is specified by a floor, a latitude and a longitude